firefox-ios/Client/Frontend/DefaultBrowserOnboarding/DefaultBrowserOnboardingViewController.swift

```diff
@@ -71,33 +71,32 @@ class DefaultBrowserOnboardingViewController: UIViewController, OnViewDismissabl
     }
 
     private lazy var titleLabel: UILabel = .build { [weak self] label in
-        label.font = DefaultDynamicFontHelper.preferredBoldFont(withTextStyle: .title1,
-                                                                size: self?.titleFontSize ?? UX.titleSize)
```

Comment:
@cwzilla the updates for the title differs from the logic that we had existing. After testing,`title1` seems to suffice in various device sizes. Can you confirm? Previously, we were showing either size `34`, `28` or `24` based on screen height.

Reference PR: https://github.com/mozilla-mobile/firefox-ios/pull/7919

Reply:
`title1` looks good to use!

this is unrelated to the font, but the button on this screen doesn't look like our standard button (it's slightly taller), can we fix it? thanks!

Reply:
Thanks for pointing that out, I will create a separate issue for that